ethsigner/signer/hashicorp/src/test/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommandTest.java

```diff
@@ -57,15 +61,37 @@ private String validCommandLine() {
         + " --signing-key-path="
         + PATH_TO_SIGNING_KEY
         + " --timeout="
-        + FIFTEEN;
+        + FIFTEEN
+        + " --tls-truststore-file="
+        + CLIENT_CERT_PFX
+        + " --tls-truststore-password-file="
+        + CLIENT_CERT_PASSWD;
   }
 
-  private String removeFieldFrom(final String input, final String fieldname) {
-    return input.replaceAll("--" + fieldname + "=.*?(\\s|$)", "");
+  private String validWithoutTlsOptionsCommandLine() {
+    return "--auth-file="
+        + THIS_IS_THE_PATH_TO_THE_FILE
+        + " --host="
+        + HTTP_HOST_COM
+        + " --port="
+        + PORT
+        + " --signing-key-path="
+        + PATH_TO_SIGNING_KEY
+        + " --timeout="
+        + FIFTEEN
+        + " --no-tls-enabled";
   }
 
-  private String modifyField(final String input, final String fieldname, final String value) {
-    return input.replaceFirst("--" + fieldname + "=.*\\b", "--" + fieldname + "=" + value);
+  private String removeFieldsFrom(final String input, final String... fieldNames) {
```

Comment:
are these in the CmdLineHelpers?

Reply:
I need to move em there.

Reply:
These are moved to CmdHelpers! Resolving convo.